provider/aws/resource.go

```diff
@@ -34,7 +34,25 @@ func (p *AWSProvider) ResourceCreate(name, kind string, params map[string]string
 	var req *cloudformation.CreateStackInput
 
 	switch s.Type {
-	case "memcached", "mysql", "postgres", "redis", "sqs":
+	case "memcached", "redis", "sqs":
+		req, err = p.createResource(s)
+	case "mysql", "postgres":
+		s.Parameters["EncryptionKey"] = ""
```

Comment:
Instead of special-casing for these two, we could parse the resource template, get its parameter list, and always set `EncryptionKey` equal to the rack's KMS key if the parameter exists.